src/pages/home/report/ReportActionItem.js

```diff
@@ -461,6 +462,24 @@ function ReportActionItem(props) {
                                     </Text>
                                 </Button>
                             )}
+                            {lodashGet(props, 'action.actionName', '') === CONST.REPORT.ACTIONS.TYPE.ACTIONABLEMENTIONWHISPER &&
+                                !lodashGet(props, 'action.originalMessage.resolution', null) && (
+                                    <ActionItemButtons
+                                        items={[
+                                            {
+                                                text: props.translate('actionableMentionWhisperOptions.invite'),
+                                                key: `${props.action.reportActionID}-actionableMentionWhisper-${CONST.REPORT.RESOLUTIONS.INVITE}`,
+                                                onPress: () => Report.resolveActionableMentionWhisper(props.report.reportID, props.action.reportActionID, CONST.REPORT.RESOLUTIONS.INVITE),
+                                                isPrimary: true,
+                                            },
+                                            {
+                                                text: props.translate('actionableMentionWhisperOptions.nothing'),
+                                                key: `${props.action.reportActionID}-actionableMentionWhisper-${CONST.REPORT.RESOLUTIONS.NOTHING}`,
+                                                onPress: () => Report.resolveActionableMentionWhisper(props.report.reportID, props.action.reportActionID, CONST.REPORT.RESOLUTIONS.NOTHING),
+                                            },
+                                        ]}
+                                    />
+                                )}
```

Comment:
Let's create a function `ReportActionsUtils.getActionableItemButtons` that takes an `action` and returns an array of the items needed to pass to the `items` prop. If we don't need any buttons for the passed `action` it'll just return an empty array.

Then we'll render the `ActionItemButtons` if the array returned from that function has size > 0.

Reply:
Ah I see what you mean, then that works with me, thanks for explaining